### PR TITLE
Edit install manual

### PR DESCRIPTION
**Pull request by tkm-kj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,13 @@ Use the development version
...
+```
+$ git clone git@github.com:krtek4/MultiPass.git
+$ cd MultiPass
+$ npm install
+$ node_modules/gulp/bin/gulp.js
+```
 2. Open the Extension panel in Chrome : Tools / Extensions
 3. Make sure the "Developer mode" checkbox is checked
-4. Click on "Load unpacked extension..." and select the folder where you cloned it
+4. Click on "Load unpacked extension..." and select the `build/chrome` folder where you cloned it
```
Comment (krtek4) [inline README.md]: The last part (`where you cloned it`) doesn't make much sense when the directory is specified

**Comment by krtek4** [comment]
Hey !

Thanks for the PR, it's true I forgot to update the documentation when I moved to `gulp` :)

Could you please just check the comment about the last step and I gladly merge your contribution :)

File: README.md (hunk 1 above)
Comment (tkm-kj) [inline README.md]: Oh..
I fix it soon :)

**Comment by krtek4** [comment]
Thanks again for the PR.

However I decided to start using npm instead of gulp to build the extension so I add to do other modifications. I also updated the documentation in the gh-pages branch.

I took inspiration in what you did though !

**Pull request closed without merging**